test/models/test_cice.py

```diff
@@ -300,3 +307,85 @@ def test_no_restart_ptr(config, cice_config_files):
         with pytest.raises(RuntimeError,
                            match="Cannot find previous restart file"):
             model.setup()
+
+
+CONFIG_WITH_COMPRESSION = {
+    "laboratory": "lab",
+    "jobname": "testrun",
+    "model": "cice",
+    "exe": "test.exe",
+    "experiment": ctrldir_basename,
+    "metadata": {"enable": False},
+    "compress_logs": True
+}
+
+
+@pytest.fixture
+def cice4_log_files():
+    """
+    Create cice log files matching those produced during ESM1.5 simulations.
+    """
+    log_names = ["ice_diag_out", "ice_diag.d", "debug.root.03",
+                 "iceout086", "iceout088", "iceout090", "iceout092",
+                 "iceout094", "iceout096", "iceout085", "iceout087",
+                 "iceout089", "iceout091", "iceout093", "iceout095"]
+    log_paths = [os.path.join(expt_workdir, name) for name in log_names]
+
+    for log_file in log_paths:
+        with open(log_file, "w") as f:
+            f.close()
+
```

Comment:
We're already making good use of the [pathlib `touch()` function](https://docs.python.org/3/library/pathlib.html#pathlib.Path.touch)

https://github.com/search?q=repo%3Apayu-org%2Fpayu+touch&type=code

Reply:
Good idea, added in 0b2fcc9